docker/env.example

```diff
@@ -1,14 +1,17 @@
 # TODO: Replace with your RPC URL
-RPC_URL=https://holesky.drpc.org
+RPC_URL=
 
 DEPLOY_ENV=LOCAL
 CHAIN_ID=17000
 ETHERSCAN_API_KEY=foobar
-LOCAL_ETHEREUM_RPC_URL=http://localhost:8545
+LOCAL_ETHEREUM_RPC_URL=http://host.docker.internal:8545
+TESTNET_RPC_URL=
 
 # FIXME: place your project's metadata URI here
 METADATA_URI=https://wavs.xyz/metadata.json
 
+FUNDED_KEY=
```

Comment:
RPC_URL & FUNDED_KEY should be set with defaults (0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80 ? ) 

also LOCAL_ETHEREUM_RPC_URL does not work by default on linux/WSL without editing /etc/hosts. So it should remain as localhost if IPV4 + v6 are supported, else 127.0.0.1

Reply:
Setting those works now on my end :)

Reply:
updated to local host , included a comment that the funded key on testnet mode should be pre-funded